include/alpaka/kernel/Traits.hpp

```diff
@@ -121,7 +121,7 @@ namespace alpaka
             TKernelFnObj const & kernelFnObj,
             vec::Vec<TDim, idx::Idx<TAcc>> const & blockThreadExtent,
             vec::Vec<TDim, idx::Idx<TAcc>> const & threadElemExtent,
-            TArgs const & ... args)
+            TArgs && ... args)
```

Comment:
Usage without `std::forward` is omehow useless.

Reply:
This were the places I overlooked on Friday short before I leave the office, thx